ccpp/suites/suite_HAFS_v0_hwrf.xml

```diff
@@ -14,12 +14,12 @@
     <subcycle loop="1">
       <scheme>GFS_suite_interstitial_rad_reset</scheme>
       <scheme>GFS_rrtmg_pre</scheme>
-      <scheme>rrtmg_sw_pre</scheme>
-      <scheme>rrtmg_sw</scheme>
-      <scheme>rrtmg_sw_post</scheme>
       <scheme>rrtmg_lw_pre</scheme>
       <scheme>rrtmg_lw</scheme>
       <scheme>rrtmg_lw_post</scheme>
+      <scheme>rrtmg_sw_pre</scheme>
```

Comment:
I am curious, why do we change the order of sw/lw radiation computation?

Reply:
Because HWRF runs lw first and sw afterwards. @mzhangw or @ligiabernardet can explain this better, hopefully.

Reply:
@climbfuji  is correct. It is the practice in HWRF/WRF.

Reply:
For what it's worth, I swapped the order and ran the HWRF regression tests against the newly created baseline with the original (HWRF) order:
```
diff --git a/ccpp/suites/suite_HAFS_v0_hwrf_thompson.xml b/ccpp/suites/suite_HAFS_v0_hwrf_thompson.xml
index 76d9aed..cb13c16 100644
--- a/ccpp/suites/suite_HAFS_v0_hwrf_thompson.xml
+++ b/ccpp/suites/suite_HAFS_v0_hwrf_thompson.xml
@@ -15,12 +15,12 @@
       <scheme>GFS_suite_interstitial_rad_reset</scheme>
       <!-- <scheme>sgscloud_radpre</scheme> -->
       <scheme>GFS_rrtmg_pre</scheme>
-      <scheme>rrtmg_lw_pre</scheme>
-      <scheme>rrtmg_lw</scheme>
-      <scheme>rrtmg_lw_post</scheme>
       <scheme>rrtmg_sw_pre</scheme>
       <scheme>rrtmg_sw</scheme>
       <scheme>rrtmg_sw_post</scheme>
+      <scheme>rrtmg_lw_pre</scheme>
+      <scheme>rrtmg_lw</scheme>
+      <scheme>rrtmg_lw_post</scheme>
       <scheme>GFS_rrtmg_post</scheme>
       <!-- <scheme>sgscloud_radpost</scheme> -->
     </subcycle>
```
The results were b4b identical.